src/runtime/rpc/rpc_host_port.cpp

```diff
@@ -157,19 +157,28 @@ std::string host_port::to_string() const
     case HOST_TYPE_IPV4:
         return fmt::format("{}:{}", _host, _port);
     case HOST_TYPE_GROUP:
-        return fmt::format("address group {}", group_host_port()->name());
+        return fmt::format("host_port group {}", group_host_port()->name());
     default:
-        return "invalid address";
+        return "invalid host_port";
     }
 }
 
+// hostname has a maximum of 253 ASCII characters
+// uint_16 port maximum is 65535
+// 253 + 1(colon) + 5 = 259
+static __thread fixed_size_buffer_pool<8, 259> bf;
+const char *host_port::get_char_str() const
```

Comment:
When `get_char_str()` would be used ? Once it is called beyond 8 times in the same thread, the content of the buffer might be updated, is this expected ?

Reply:
e.g.
`sprintf("%s", get_char_str())`
`strcpy(dest, get_char_str())`

If we just have a funcation return temporary vaule not enough.

Initially, I thought that a buffer for storing one string would be sufficient.
Later I found `fixed_size_buffer_pool` from class `rpc_address` and thought it was more suitable.
Updated buffer not have influence on use.

Reply:
OK. Please add some comments to illustrate our special scenarios like `log_prefix()`:
![企业微信截图_f370bcab-9324-4855-9da9-3d2d9de2b155](https://github.com/apache/incubator-pegasus/assets/743379/3c0f89b7-ff14-46d1-a287-86fb2ff76b60)

Otherwise,  using `get_char_str` by mistake may lead to some bugs:
```
// The size of hosts are more than 8.
std::vector<host_port> hosts;
......
std::vector<const char *> strs;
for (auto &e: hosts) {
    strs.push_back(e.get_char_str());
}
```

Reply:
I still doubt the necessarity of introducing it, what will happen if using:
```
fmt::printf("{}", to_std_string())
strcpy(dest, to_std_string().c_str())
```